api-docs.js

```diff
@@ -55,6 +55,25 @@ const apiDoc = {
         },
       },
     },
+    '/new-seed': {
```

Comment:
It is not secure to return the seed, and there is no way to use this seed in the wallet. I don't understand why we would add such method.